src/common/ManagedCommon/Position.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace ManagedCommon
+{
+    public enum Position
+    {
+        Cursor,
+        Primary,
+        Focus,
+    }
+}
```

Comment:
Is this worth moving closer to usage? Again, I initially made this in the common project so it could be reused by other projects but I'm happy to move it if it's not appropriate.

Reply:
@addrum
it's ok to move it to the PT Run project, adding thing to common lib by default is not actually helping maintaining the common libs.

Reply:
It would not be possible to move it to PT Run project(it is still named PowerLauncher) as we want to share enum between settings and PT Run.